root/edit/SubscribedEdits.js

```diff
@@ -27,6 +27,13 @@ const SubscribedEdits = ({
   <Layout fullWidth title={l('Edits for Your Subscribed Entities')}>
     <div id="content">
       <h1>{l('Edits for Your Subscribed Entities')}</h1>
+
+      <p>
+        {l(`This page lists edits linked to entities you are directly
+            subscribed to, as well as edits linked to all entities which are
```

Comment:
I think "all" is implied here. It sounds better to me without it because the first part of the sentence says "linked to entities" but this one says "linked to all entities" which makes it sound like there's a difference?

```suggestion
            subscribed to, as well as edits linked to entities which are
```